collect_app/src/main/java/org/odk/collect/android/activities/GeoPointOsmMapActivity.java

```diff
@@ -519,7 +520,7 @@ public void onCancel(DialogInterface dialog) {
         if (myLocationOverlay.getMyLocation() != null) {
             zoomLocationButton.setEnabled(true);
             zoomLocationButton.setBackgroundColor(Color.parseColor("#50cccccc"));
-            zoomLocationButton.setTextColor(Color.parseColor("#ff333333"));
+            zoomLocationButton.setTextColor(new ThemeUtils(this).getPrimaryTextColor());
```

Comment:
`themeUtils` is already initialized in `CollectAbstractActivity`. So that can be reused